### PR TITLE
K8S-240 Rename secret for neo4j dataset secret

### PR DESCRIPTION
**Pull request by thibveni**

### CODE REVIEW

File: charts/neo4j-internal/values.preprod.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ neo4j-standalone:
         command: ['/data/docker-neo4j/scripts/dataset-download.sh', '-p', '/data']
         envFrom:
           - secretRef:
-              name: neo4j-preprod-nexus-dataset # imports NEXUS_TOKEN
+              name: neo4j-nexus-dataset-secret # imports NEXUS_TOKEN
```
Comment (thibveni) [inline charts/neo4j-internal/values.preprod.yaml:59]: I confirm that both secret are existing and with correct values on dev and preprod namespace

**Comment by thibveni** [comment]
@enys  Do we agree that i will not do any change on prod ? there is no kyverno policy to add for this one and we accept to have a different name for the secret ? same for saas

File: charts/neo4j-internal/values.preprod.yaml (hunk 1 above)
Comment (thibveni) [reply charts/neo4j-internal/values.preprod.yaml:59]: and i also confirm that kyverno policy worked and it duplicate secret from preprod to dev

**Comment by enys** [comment]
> @enys Do we agree that i will not do any change on prod ? there is no kyverno policy to add for this one and we accept to have a different name for the secret ? same for saas

Good catch, agreed.

**Review by enys — APPROVED** [review]

**Pull request merged**